website/www/site/layouts/_default/baseof.html

```diff
@@ -20,13 +20,12 @@
   <div class="body__contained no__padding">
     {{ block "hero-section" . }}{{ end }}
     {{ block "ctas-section" . }}{{ end }}
-    {{ block "pillars-section" . }}{{ end }}
-    {{ block "playground-section" . }}{{ end }}
```

Comment:
I don't see playground section on the staging. It seems that we do not need remove this line

Reply:
done